api/machines.go

```diff
@@ -27,6 +30,10 @@ import (
 	"github.com/coreos/fleet/schema"
 )
 
+var (
+	metadataPathRegex = regexp.MustCompile("^/([0-9a-f]{32})/metadata/([A-Za-z0-9_.-]+$)")
```

Comment:
Gotcha, will change.

On Fri, Jan 30, 2015 at 5:42 PM, Brian Waldon <notifications@github.com>
wrote:

> In api/machines.go
> <https://github.com/coreos/fleet/pull/1077#discussion_r23878227>:
>
> > @@ -27,6 +30,10 @@ import (
> >  	"github.com/coreos/fleet/schema"
> >  )
> >
> > +var (
> > +	metadataPathRegex = regexp.MustCompile("^/([0-9a-f]{32})/metadata/([A-Za-z0-9_.-]+$)")
>
> fleet's machines don't necessarily have to use systemd's machine-id
> concept to identify themselves. It can be overridden in config. This needs
> to be an arbitrary string.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/fleet/pull/1077/files#r23878227>.
>